### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by devhkr**
As @Moshe-Immerman brought [the idea](https://github.com/veggiemonk/awesome-docker/pull/380#issuecomment-314560929) updating the PR rules, I updated them here.

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -20,8 +20,16 @@
...
-RULES (because you probably didn't read the guidelines): 
-* ORDER: The entries are in lexical order (English letter order) 
-* CREDIT: Each entry must contains the name of the author with a link to its github account!
-* LINK: the link should be to the source code (https://github/account/project) not the website (https://myproject.io <- no!)
-* MONEY: indicates if one needs a paying subscription to use that project you could just add ($$$) next to the name ([my project - $$$] (link to my project) )
+**RULES (because you probably didn't read the guidelines):**
+- FORMAT
+  - Use the following format: `[Name](Link) - Description.`
+  - Name should be the title of the project
+  - Link should be to the GitHub source code, not to the website
```
Comment (agebhar1) [inline pull_request_template.md]: is it good english? ... :thinking:

File: pull_request_template.md [hunk 2]
```diff
@@ -20,8 +20,16 @@
...
+  - Description should end with a full stop/period
+  - After the description, add the name of the author with a link to its GitHub account like `By [@author](GitHub account)`
```
Comment (agebhar1) [inline pull_request_template.md]: the Travis CI job checks for duplicated links - we should mention the markdown named link (?) at the end of the file

File: pull_request_template.md (hunk 1 above)
Comment (devhkr) [inline pull_request_template.md]: You're right, it sounds bit weird. Maybe like this?
- Link should be a GitHub repository, not a website

File: pull_request_template.md (hunk 2 above)
Comment (devhkr) [inline pull_request_template.md]: Good point @agebhar1 

Yes, we should mention that.

File: pull_request_template.md (hunk 1 above)
Comment (agebhar1) [inline pull_request_template.md]: :+1:

Comment (dmitrytokarev) [inline pull_request_template.md]: link to something...
so `Link should be to a GitHub repository, not to a website`
(disclaimer English is my second language but i have used it for 25 years)

File: pull_request_template.md (hunk 2 above)
Comment (devhkr) [inline pull_request_template.md]: @agebhar1 added the notice under the rules section

File: pull_request_template.md (hunk 1 above)
Comment (devhkr) [inline pull_request_template.md]: @dmitrytokarev 

I've read the sentence several times. To be more perfect,
`Link should be pointing to a GitHub repository, not to a website`

**Pull request merged**